packages/ace-linters/type-converters/common-converters.ts

```diff
@@ -1,12 +1,13 @@
-import {Ace, Range as AceRange} from "ace-code";
+import {Ace} from "ace-code";
```

Comment:
should this be `import type {Ace}` ?

Reply:
It doesn't matter a lot here, and `esbuild-loader` didn't understand `import type` syntax, so I changed it. Right now we could start using it again due to `swc-loader` is normally transpiling ts code